odpf/shield/v1/shield.proto

```diff
@@ -333,22 +333,27 @@ message GroupRequestBody {
   string name = 1 [(validate.rules).string.pattern = "^[A-Za-z0-9_-]+$"];
   string slug = 2;
   google.protobuf.Struct metadata = 3;
+  string org_id = 4;
 }
 
 message CreateGroupRequest {
   GroupRequestBody body = 1;
 }
 
 message Group {
+  reserved 6;
+  reserved "org_id";
+
   string id = 1;
   string name = 2 [(validate.rules).string.pattern = "^[A-Za-z0-9_-]+$"];
   string slug = 3;
   repeated User admins = 4;
   int32 member_count = 5;
-  string org_id = 6;
+  // string OBSOLETE_org_id = 6;  [OBSOLETE]
```

Comment:
@krtkvrm Since the shield GRPC version is not yet shipped. I think it is ok to make a breaking change and not reserve it.

Reply:
Had to add it as CI was showing Breaking Changes when I just replaced it with `Organization organization = 6`
Ref: https://github.com/odpf/proton/runs/4075364158?check_suite_focus=true#step:4:10

Reply:
We can merge it irrespective of that. Lets also see if we can enable disable breaking change ci specific to certain projects.